auth/README.md

```diff
@@ -263,13 +263,27 @@ country code and phone number input fields. The user is still able to edit the n
 ```java
 // Use a Bundle to hold the default number, and pass it to the Builder via setParams:
 Bundle params = new Bundle();
-params.putString(AuthUI.EXTRA_DEFAULT_PHONE, "+12345678901");
+params.putString(EXTRA_DEFAULT_PHONE, "+123456789");
```

Comment:
We should leave the `AuthUI.` otherwise developers won't know where this constant comes from

Reply:
Good point, updated now.